meson.build

```diff
@@ -59,34 +59,28 @@ message('has_exceptions: @0@'.format(has_exceptions))
 
 # GCC or Clang
 if is_gcc or is_clang
-	add_project_arguments(
-		'-march=native',
-		'-fvisibility=hidden',
-		'-fvisibility-inlines-hidden',
-		language: 'cpp'
-	)
+	devel_args += '-march=native'
 	if is_release
-		add_project_arguments(
+		devel_args += [
 			'-fdata-sections',
 			'-ffunction-sections',
 			'-Wl,--gc-sections',
 			'-Wl,-s',
 			'-mfma',
-			language: 'cpp'
-		)
+		]
 	endif
 endif
 
 
 # GCC
 if is_gcc
-	add_project_arguments(
+	devel_args += [
 		'-fmax-errors=5',
 		'-Wno-init-list-lifetime',
-		language: 'cpp'
-	)
+		'-DTOML_EXTERN_TEMPLATES=0'
```

Comment:
why? it was never necessary here before. `TOML_EXTERN_TEMPLATES` is intended to solve a different problem.